include/mls/messages.h

```diff
@@ -455,12 +480,52 @@ struct Sender
   TLS_SERIALIZABLE(sender_type, sender)
 };
 
+struct MLSPlaintextTBS
```

Comment:
Can `MLSPlaintextTBS` and `MLSPlaintextTBM` move into `messages.cpp`?  It seems like they're not needed outside of signing/MAC operations.

Reply:
Likewise for `MLSCiphertextContent` and `MLSCiphertextContentAAD` below.

Reply:
Yep, will do.